controllers/dashboardController.js

```diff
@@ -0,0 +1,63 @@
+
+const jwt = require("jsonwebtoken")
+
+const config = require("../config")
+const conferences = require("../lib/conferences")
+const { decrypt } = require("../lib/crypto")
+
+module.exports.get = async (req, res) => {
+    const token = req.params.token
+
+    if (!token) {
```

Comment:
un flash message serait sympa, par exemple :
```suggestion
    if (!token) {
            req.flash("error", "La conférence a expiré. Vous pouvez recréer une conférence.")

```
(en supposant que l'utilisateur arrive sur cette url sans token, chepa trop comment... On sait jamais, dans le doute soyons poli)